packages/server/lib/controllers/connection.controller.ts

```diff
@@ -77,7 +77,7 @@ class ConnectionController {
                     content: 'Unknown connection'
                 });
                 const logCtx = await logContextGetter.create(
-                    { id: String(activityLogId), operation: { type: 'token' }, message: 'Get connection web' },
```

Comment:
I'm not sure if I completely agree with getting rid of `token` and replacing it with `auth`. There is a difference from doing an `authorization` (auth) call vs. fetching a token or a token refresh failure. 

Also feel like this is partially a product decision to what the end user sees.